sig-api-machinery/annual-report-2023.md

```diff
@@ -48,24 +53,10 @@
     - [2896 - OpenAPI V3](https://github.com/kubernetes/enhancements/tree/master/keps/sig-api-machinery/2896-openapi-v3) - v1.27
     - [365 - Paginated API Lists](https://github.com/kubernetes/enhancements/tree/master/keps/sig-api-machinery/365-paginated-lists) - v1.29
 
-## [Subprojects](https://git.k8s.io/community/sig-api-machinery#subprojects)
```

Comment:
I reviewed the KEP list against my personal notes.

- Generic Control Plane started - https://github.com/kubernetes/enhancements/pull/4052 - alpha 1.27 (I believe)
- Move Storage Version Migrator in-tree - https://github.com/kubernetes/enhancements/pull/4240 - alpha 1.29